### PR TITLE
structure: subdir ibmcloud ipi

### PR DESCRIPTION
**Pull request by sean-freeman**
Create sub-directory for IPI under IBM Cloud infrastructure platform vendor-specific documentation. This includes re-writing of all filename references within the repository and fixes to chapter titles/introductions to follow sentence format used elsewhere (i.e. AWS, AZ).

> **Output structure after subdir creation**

- Installing on IBM Cloud
    - Installer-provisioned infrastructure
        - Installing a cluster with customizations on IBM Cloud
        - Installing a cluster with network customizations on IBM Cloud
        - Installing a cluster with an existing VPC network on IBM Cloud
        - Installing a private cluster with an existing VPC network on IBM Cloud
        - Installing a cluster with a restricted VPC network on IBM Cloud

Version(s):
4.17, do not cherry pick as this file is likely to have changed between each version

Issue:
- None (GH or Jira), this is community author contribution

QE review:
- [ ] QE has approved this change.

Additional information:
- n/a

### CODE REVIEW

**Comment by sean-freeman** [comment]
@kalexand-rh @bscott-rh @sheriff-rh Mentioning you all, as GH Issue template requests @openshift/team-documentation [which does not work] is mentioned after PR is raised.

**Comment by sean-freeman** [comment]
@kalexand-rh @bscott-rh @sheriff-rh Can you please review this PR? I would prefer to avoid merge conflicts which is more likely the longer this PR is open.

**Comment by bscott-rh** [comment]
I am holding this PR until I can discuss with the docs program manager. This PR has the potential to affect the publishing system for the new docs site due to the topic map changes. 

/ok-to-test

**Comment by sean-freeman** [comment]
@bscott-rh If changes are made to GCP structure (as described in another PR) in the new docs site, I would request this PR to be accepted also. Otherwise this conflicts with vendor neutrality

File: welcome/learn_more_about_openshift.adoc [hunk 1]
```diff
@@ -82,7 +82,7 @@ a| If your cluster uses user-provisioned infrastructure, and the cluster does no
 * xref:../installing/installing_aws/upi/installing-restricted-networks-aws.adoc#installing-restricted-networks-aws[{aws-first}]
 * xref:../installing/installing_gcp/installing-restricted-networks-gcp.adoc#installing-restricted-networks-gcp[{gcp-short}]
 * xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[{vmw-short}]
-* xref:../installing/installing_ibm_cloud/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}]
+* xref:../installing/installing_ibm_cloud/ipi/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}]
```
[the remark this answers is not in the document]
Comment (sean-freeman) [reply welcome/learn_more_about_openshift.adoc:85]: Irrelevant, vale identified URI subdir and not a typo.

File: rosa_architecture/index.adoc [hunk 2]
```diff
@@ -114,7 +114,7 @@ ifndef::openshift-origin[]
...
-xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[{vmw-short}], xref:../installing/installing_ibm_cloud/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}], xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[{ibm-z-name} and {ibm-linuxone-name}], xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power[{ibm-power-name}],
+xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[{vmw-short}], xref:../installing/installing_ibm_cloud/ipi/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}], xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[{ibm-z-name} and {ibm-linuxone-name}], xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power[{ibm-power-name}],
  or
```
[the remark this answers is not in the document]
Comment (sean-freeman) [reply rosa_architecture/index.adoc:118]: Irrelevant, vale identified URI subdir and not a typo.

File: installing/overview/installing-preparing.adoc [hunk 3]
```diff
@@ -77,7 +77,7 @@ If you use a user-provisioned installation method, you can configure a proxy for
...
...icted-networks-bare-metal.adoc#installing-restricted-networks-bare-metal[bare metal]. You can also install a cluster into a restricted network using installer-provisioned infrastructure by following detailed instructions for xref:../../installing/installing_aws/ipi/installing-restricted-networks-aws-installer-provisioned.adoc#installing-restricted-networks-aws-installer-provisioned[AWS], xref:../../installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc#installing-restricted-networks-gcp-installer-provisioned[GCP], xref:../../installing/installing_ibm_cloud/ipi/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}], xref:../../installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.adoc#installing-restricted-networks-nutanix-installer-provisioned[Nutanix], xref:../../installing/installing_openstack/installing-openstack-installer-restricted.adoc#installing-openstack-installer-restricted[{rh-openstack}], and xref:../../installing/installing_vsphere/ipi/installing-restricted-networks-installer-provisioned-vsphere.adoc#installing-restricted-networks-installer-provisioned-vsphere[vSphere].
```
[the remark this answers is not in the document]
Comment (sean-freeman) [reply installing/overview/installing-preparing.adoc:80]: Irrelevant, vale identified URI subdir and not a typo.

File: installing/installing_ibm_cloud/ipi/installing-ibm-cloud-restricted.adoc [hunk 4]
```diff
@@ -1,37 +1,40 @@
...
 [id="installing-ibm-cloud-restricted"]
-= Installing a cluster on {ibm-cloud-title} in a restricted network
+= Installing a cluster with a restricted VPC network on {ibm-cloud-title}
 :context: installing-ibm-cloud-restricted
 
 toc::[]
 
-In {product-title} {product-version}, you can install a cluster in a restricted network by creating an internal mirror of the installation release content that is accessible to an existing Virtual Private Cloud (VPC) on {ibm-cloud-name}.
+In {product-title} version {product-version}, you can install a
+cluster, with a restricted Virtual Private Cloud (VPC) network, on infrastructure that the installation program provisions on {ibm-cloud-name} using an internal mirror of installation media from an existing VPC on {ibm-cloud-name}; optionally creation of a restricted cluster (aka. disconnected cluster).
```
[the remark this answers is not in the document]
Comment (sean-freeman) [reply installing/installing_ibm_cloud/ipi/installing-ibm-cloud-restricted.adoc:10]: Will not fix, leaving content itself as-is.

**Pull request closed without merging**